services/src/main/java/org/exoplatform/officeonline/rest/WOPIResource.java

```diff
@@ -798,66 +823,65 @@ private Response delete(EditorConfig config) {
+                     .entity("{\"Name\": \"\", \"Url\": \"\"}")
                      .type(MediaType.APPLICATION_JSON)
                      .build();
     } catch (FileLockedException e) {
       if (LOG.isDebugEnabled()) {
         LOG.debug("File locked error for PutRelativeFile", e);
       }
       return Response.status(Status.CONFLICT)
-                     .entity("{\"error\": \"" + e.getMessage() + "\"}")
+                     .entity("{\"Name\": \"\", \"Url\": \"\"}")
                      .header(LOCK, e.getLockId() != null ? e.getLockId() : "")
                      .type(MediaType.APPLICATION_JSON)
                      .build();
     } catch (UpdateConflictException e) {
       if (LOG.isDebugEnabled()) {
         LOG.debug("Update conflict for PutRelativeFile", e);
       }
-      return Response.status(Status.CONFLICT)
-                     .entity("{\"error\": \"" + e.getMessage() + "\"}")
-                     .type(MediaType.APPLICATION_JSON)
-                     .build();
+      return Response.status(Status.CONFLICT).entity("{\"Name\": \"\", \"Url\": \"\"}").type(MediaType.APPLICATION_JSON).build();
     } catch (FileNotFoundException e) {
       if (LOG.isDebugEnabled()) {
         LOG.debug("File not found for PutRelativeFile", e);
       }
-      return Response.status(Status.NOT_FOUND)
-                     .entity("{\"error\": \"" + e.getMessage() + "\"}")
-                     .type(MediaType.APPLICATION_JSON)
-                     .build();
+      return Response.status(Status.NOT_FOUND).entity("{\"Name\": \"\", \"Url\": \"\"}").type(MediaType.APPLICATION_JSON).build();
     } catch (FileExtensionNotFoundException e) {
       if (LOG.isDebugEnabled()) {
         LOG.debug("File extension not found for PutRelativeFile", e);
       }
       return Response.status(Status.INTERNAL_SERVER_ERROR)
-                     .entity("{\"error\": \"" + e.getMessage() + "\"}")
+                     .entity("{\"Name\": \"\", \"Url\": \"\"}")
                      .type(MediaType.APPLICATION_JSON)
                      .build();
     } catch (PermissionDeniedException e) {
       if (LOG.isDebugEnabled()) {
         LOG.debug("Cannot create new file based on existing one in specific mode.", e);
       }
-      return Response.status(Status.FORBIDDEN)
-                     .entity("{\"error\": \"" + e.getMessage() + "\"}")
-                     .type(MediaType.APPLICATION_JSON)
-                     .build();
+      return Response.status(Status.FORBIDDEN).entity("{\"Name\": \"\", \"Url\": \"\"}").type(MediaType.APPLICATION_JSON).build();
     } catch (RepositoryException e) {
       LOG.error("Cannot create new file based on existing one in specific mode.", e);
       return Response.status(Status.INTERNAL_SERVER_ERROR)
-                     .entity("{\"error\": \"Cannot create new file based on existing one in specific mode.\"}")
+                     .entity("{\"Name\": \"\", \"Url\": \"\"}")
```

Comment:
How this will work? We don't send any readable error message into the response, how then to understand that an error reported by an user (enduser or admin) is connected with an one in the logs?
We need JSON with "error" element in the root, then with "message" for a human readable text and details fields if applicable.

Reply:
Fixed: https://github.com/exo-addons/office-online/pull/12/commits/1dbc50655f9e4081bde0523b78ad7e9e73bd36bd